### PR TITLE
Update Topics “criminal-justice”

### PR DESCRIPTION
**Pull request by vwilliamsn**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by kewitham** [comment]
@vwilliamsn this should be good to go now! holding off merging pending your review.

**Review by kewitham — APPROVED** [review]

**Comment by vwilliamsn** [comment]
LGTM! Thank @kewitham for all your help!!!

**Pull request merged**
